screens/auth/WelcomeScreen.js

```diff
@@ -64,8 +64,6 @@ class WelcomeScreen extends Component {
                 />
                 <SafeAreaView
                     style={{ flex: 1, backgroundColor: Colors.accentLight }}>
-                    <LanguageFlags lang={lang} />
```

Comment:
Where does it show?

Reply:
I think this delete was a mistake?

Reply:
Its added after the view.. Component towards the end.. 
Did that so that absolute positioning respect the SafeareaView..